WaterDock-2.0_Basics.md

```diff
@@ -0,0 +1,76 @@
+**This file is a more detailed version of README.md.**
+N.B. The command line commands in this text file work for Mac but may not work for Windows and Linux. You might need to look up the equivalent commands for these operating systems.

```

Comment:
A thought for a future iteration, most users will be using a Linux machine (in Windows you can just tell people to use WSL). What things here are actually MacOS specific?

Reply:
Ah I did not know that Linux and MacOS were so similar  - I thought at the time maybe the commands for adding vina to the PATH might be different between Linux and MacOS but I now doubt this. Would be still different for windows though I believe. 

Maybe it would be less confusing to add an edited version of this instruction to that part of file that is talking about adding vina to your PATH.

Reply:
The way PATH is added here is indeed completely different from how you would normally do this in Linux. (I'm not familiar with how Macs handle their local shell, I assume it's not feasible to just add an export PATH call in one's own .bashrc or .zshrc file?)

One potential approach here would be to just say "vina must be in your PATH" (which is what vina itself does for it's own instructions). It depends on how detailed you want the instructions to be. @philbiggin might have a better idea of how much detail is necessary here.